### PR TITLE
Created burden denominator vars and added all gross rent vars

### PR DESCRIPTION
**Pull request by sdiby**
@ptatian new variables are ready for your review.

- Created vars for renter and owner housing units where cost burden is computed. By age and income. To serve as denominator for rent burden variables.

- Also added gross rent variables according to ACS categories (I had previously combined multiple)

### CODE REVIEW

**Comment by ptatian** [comment]
Thanks, @sdiby. I will review.

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 1]
```diff
@@ -685,42 +729,77 @@
...
+
+  mGrossRentLT100_&_years. = B25063m3;
+  mGrossRent100_149_&_years. = B25063m4;
+  mGrossRent150_199_&_years. = B25063m5;
+  mGrossRent200_249_&_years. = B25063m6;
+  mGrossRent250_299_&_years. = B25063m7;
+  mGrossRent300_349_&_years. = B25063m8;
+  mGrossRent350_349_&_years. = B25063m9;
+  mGrossRent400_449_&_years. = B25063m10;
+  mGrossRent450_499_&_years. = B25063m11
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Missing ;

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 2]
```diff
@@ -685,42 +729,77 @@
...
+  mGrossRent750_799_&_years. = B25063m17;
+  mGrossRent800_899_&_years. = B25063m18;
+  mGrossRent900_999_&_years. = B25063m19;
+  mGrossRent1000_1249_&_years. = B25063m20;
+  mGrossRent1250_1499_&_years. = B25063m21;
+  mGrossRent1500_1999_&_years. = B25063m22;
+  mGrossRent2000_2499_&_years. = B25063m23;
+  mGrossRent2500_2999_&_years. = B25063m24;
+  mGrossRent3000_3499_&_years. = B25063m25;
+  mGrossRentGT3500&_years. = B25063m26;
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Missing _ in var name

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 3]
```diff
@@ -641,23 +641,67 @@
...
+
+  GrossRentLT100_&_years. = B25063e3;
+  GrossRent100_149_&_years. = B25063e4;
+  GrossRent150_199_&_years. = B25063e5;
+  GrossRent200_249_&_years. = B25063e6;
+  GrossRent250_299_&_years. = B25063e7;
+  GrossRent300_349_&_years. = B25063e8;
+  GrossRent350_349_&_years. = B25063e9;
+  GrossRent400_449_&_years. = B25063e10;
+  GrossRent450_499_&_years. = B25063e11
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Missing ;

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 4]
```diff
@@ -641,23 +641,67 @@
...
+  GrossRent750_799_&_years. = B25063e17;
+  GrossRent800_899_&_years. = B25063e18;
+  GrossRent900_999_&_years. = B25063e19;
+  GrossRent1000_1249_&_years. = B25063e20;
+  GrossRent1250_1499_&_years. = B25063e21;
+  GrossRent1500_1999_&_years. = B25063e22;
+  GrossRent2000_2499_&_years. = B25063e23;
+  GrossRent2500_2999_&_years. = B25063e24;
+  GrossRent3000_3499_&_years. = B25063e25;
+  GrossRentGT3500&_years. = B25063e26;
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Missing _ in var name

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 5]
```diff
@@ -685,42 +729,77 @@
...
+  mIncmByRenterCst_LT10K_&_years. = %moe_sum( var=B25074m3 B25074m4 B25074m5 B25074m6 B25074m7 B25074m8 B25074m9);
+  mIncmByRenterCst_10_19K_&_years. = %moe_sum( var=B25074m12 B25074m13 B25074m14 B25074m15 B25074m16 B25074m17 B25074m18);
+  mIncmByRenterCst_20_34K_&_years. = %moe_sum( var=B25074m21 B25074m22 B25074m23 B25074m24 B25074m25 B25074m26 B25074m27);
+  mIncmByRenterCst_35_49K_&_years. = %moe_sum( var=B25074m30 B25074m31 B25074m32 B25074m33 B25074m34 B25074m35 B25074m36);
+  mIncmByRenterCst_50_74K_&_years. = %moe_sum( var=B25074m39 B25074m40 B25074m41 B25074m42 B25074m43 B25074m44 B25074m45);
+  mIncmByRenterCst_75_99K_&_years. = %moe_sum( var=B25074m48 B25074m49 B25074m50 B25074m51 B25074m52 B25074m53 B25074m54);
+  mIncmByRenterCst_GT100K_&_years. = %moe_sum( var=B25074m57 B25074m58 B25074m59 B25074m60 B25074m61 B25074m62 B25074m63);
+
+  mAgeByRenterCst_15_24_&_years. = %moe_sum( var=B25074m3 B25074m4 B25074m5 B25074m6 B25074m7);
+  mAgeByRenterCst_25_34_&_years. = %moe_sum( var=B25074m10 B25074m11 B25074m12,B25074m13 B25074m14);
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Stray comma in %moe_sum() var list

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 6]
```diff
@@ -685,42 +729,77 @@
...
+  mAgeByRenterCst_35_64_&_years. = %moe_sum( var=B25074m17 B25074m18 B25074m19 B25074m20 B25074m21);
+  mAgeByRenterCst_65Over_&_years. = %moe_sum( var=B25074m24 B25074m25 B25074m26 B25074m27 B25074m28);
+
+  mAgeByOwnerCst_15_24_&_years. = %moe_sum( var=B25093m3 B25093m4 B25093m5 B25093m6 B25093m7);
+  mAgeByOwnerCst_25_34_&_years. = %moe_sum( var=B25093m10 B25093m11 B25093m12,B25093m13 B25093m14);
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: Stray comma in %moe_sum() var list

File: Macros/ACS_summary_geo_source_bg_vars.sas [hunk 7]
```diff
@@ -858,20 +993,8 @@
...
-      mGrossRent1000_1249_&_years. = "Renter-occupied housing units where gross rent is $1000 to $1249, MOE, &_years_dash "
-      mGrossRent1250_1499_&_years. = "Renter-occupied housing units where gross rent is $1250 to $1499, MOE, &_years_dash "
-      mGrossRent1500_1999_&_years. = "Renter-occupied housing units where gross rent is $1500 to $1999, MOE, &_years_dash "
-      mGrossRent2000_2499_&_years. = "Renter-occupied housing units where gross rent is $2000 to $2499, MOE, &_years_dash "
-      mGrossRent2500_2999_&_years. = "Renter-occupied housing units where gross rent is $2500 to $2999, MOE, &_years_dash "
-      mGrossRent3000_3499_&_years. = "Renter-occupied housing units where gross rent is $3000 to $3499, MOE, &_years_dash "
-      mGrossRentGT3500_&_years. = "Renter-occupied housing units where gross rent is greater than $3500, MOE, &_years_dash "
   medianhomevalue_&_years. = "Median value of owner-occupied housing units ($),&_years_dash"
   mmedianhomevalue_&_years.="Median value of owner-occupied housing units ($), MOE, &_years_dash"
-
```
Comment (ptatian) [inline Macros/ACS_summary_geo_source_bg_vars.sas]: There was an error that seemed to be generated by this label statement, but I couldn't see what the cause was. Please check again when you re-run. 

```
ERROR 22-322: Syntax error, expecting one of the following: a name, a quoted string, a numeric constant, 
              a datetime constant, a missing value, INPUT, PUT.  
```

**Comment by sdiby** [comment]
@ptatian I've made the changes and re-ran the program--it seems to run okay and is ready for your review.

**Comment by ptatian** [comment]
Thanks, @sdiby. Yes, it looks like it runs properly now. One thing I forgot, though. We should change the   _revisions=_ message in the four summary programs so that it says something about what is new in this update of the files. Something about adding the cost burden denominators, I suppose. Once you do that, I think it is ready to finalize everything.

**Comment by sdiby** [comment]
Thanks @ptatian . I've edited the revisions= message in the four summary programs. If this looks okay (i.e. isn't too long), I'll go ahead and merge the request.

**Comment by ptatian** [comment]
@sdiby Looks good. Please merge and delete the branch, copy programs to L:, and batch submit. When done, you can close issue #79.  I can review the BridgePk pull request once the ACS data are run. 

Thank you!

**Comment by sdiby** [comment]
I've batch submitted on L: and I'm closing the issue now. Please feel free to review the BridgePk.

Thanks @ptatian !!

**Pull request merged**